Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -84,7 +84,10 @@ bool NANDImporter::ReadNANDBin(const std::string& path_to_bin,
   {
     const std::string otp_dump_path = get_otp_dump_path();
     if (otp_dump_path.empty())
+    {
+      PanicAlertFmtT("A NAND backup can't be decrypted without the associated keys.");
```

Comment:
Could we put this in Java_org_dolphinemu_dolphinemu_utils_WiiUtils_importNANDBin instead? Having a panic alert here seems to defeat the point of having get_otp_dump_path to avoid UI interactions in the core logic.

Reply:
I figured this fit with the other panic alerts in this file, but sure, I don't mind moving it.